Amazon.QLDB.Driver.IntegrationTests/AsyncStatementExecutionTests.cs

```diff
@@ -0,0 +1,1053 @@
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+namespace Amazon.QLDB.Driver.IntegrationTests
+{
+    using Amazon.QLDB.Driver.IntegrationTests.utils;
+    using Amazon.QLDBSession;
+    using Amazon.QLDBSession.Model;
+    using Amazon.IonDotnet.Builders;
+    using Amazon.IonDotnet.Tree;
+    using Amazon.IonDotnet.Tree.Impl;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.Threading.Tasks;
+    
+    [TestClass]
+    public class AsyncStatementExecutionTests
+    {
+        private static readonly IValueFactory ValueFactory = new ValueFactory();
+        private static AmazonQLDBSessionConfig amazonQldbSessionConfig;
+        private static IntegrationTestBase integrationTestBase;
+        private static AsyncQldbDriver qldbDriver;
+
+        [ClassInitialize]
+        public async static Task SetUp(TestContext context)
+        {
+            // Get AWS configuration properties from .runsettings file.
+            string region = context.Properties["region"].ToString();
+            
+            amazonQldbSessionConfig = IntegrationTestBase.CreateAmazonQLDBSessionConfig(region);
+            integrationTestBase = new IntegrationTestBase(Constants.LedgerName, region);
+
+            integrationTestBase.RunForceDeleteLedger();
+
+            integrationTestBase.RunCreateLedger();
+            qldbDriver = integrationTestBase.CreateAsyncDriver(amazonQldbSessionConfig);
+
+            // Create table.
+            var query = $"CREATE TABLE {Constants.TableName}";
+
+            var count = await qldbDriver.Execute(async txn =>
+            {
+                var result = await txn.Execute(query);
+
+                var count = 0;
+                await foreach (var row in result)
+                {
+                    count++;
+                }
+                return count;
+            });
+
+
+            Assert.AreEqual(1, count);
+
+            var result = await qldbDriver.ListTableNamesAsync();
+            foreach (var row in result)
+            {
+                Assert.AreEqual(Constants.TableName, row);
+            }
+        }
+
+        [ClassCleanup]
+        public static void ClassCleanup()
+        {
+            integrationTestBase.RunDeleteLedger();
+            qldbDriver.Dispose();
+        }
+
+        [TestCleanup]
+        public async Task TestCleanup()
+        {
+            // Delete all documents in table.
+            await qldbDriver.Execute(async txn => await txn.Execute($"DELETE FROM {Constants.TableName}"));
+        }
+
+        [TestMethod]
+        public async Task Execute_DropExistingTable_TableDropped()
+        {
+            // Given.
+            var create_table_query = $"CREATE TABLE {Constants.CreateTableName}";
```

Comment:
I know you ported this from the non-asyc tests, but the variable name convention is incorrect. It should be camel case.

e.g. createTableQuery.

Please look over this file and make the updates

Reply:
This doesn't seem to been updated

Reply:
Addressed